Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb

```diff
@@ -292,197 +141,40 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'fbprophet' is not defined",
```

Comment:
Missing virtual environment. Means every time someone downloads the file, it's broken